packages/prisma/schema.prisma

```diff
@@ -254,27 +257,43 @@ model User {
   routingForms         App_RoutingForms_Form[] @relation("routing-form")
   verifiedNumbers      VerifiedNumber[]
   hosts                Host[]
-  organizationId       Int?
-  organization         Team?                   @relation("scope", fields: [organizationId], references: [id], onDelete: SetNull)
   accessCodes          AccessCode[]
   // Linking account code for orgs v2
   //linkedByUserId       Int?
   //linkedBy             User?                   @relation("linked_account", fields: [linkedByUserId], references: [id], onDelete: Cascade)
   //linkedUsers          User[]                  @relation("linked_account")*/
 
   // Used to lock the user account
-  locked Boolean @default(false)
+  locked      Boolean      @default(false)
+  orgProfiles OrgProfile[]
 
   @@unique([email])
   @@unique([email, username])
-  @@unique([username, organizationId])
```

Comment:
We will drop `organizationId` in separate deployment. If we remove it now, there would be `username` conflicts in the table.